azurerm/internal/services/web/app_service.go

```diff
@@ -1424,10 +1424,13 @@ func expandAppServiceLogs(input interface{}) web.SiteLogsConfigProperties {
 		httpLogsConfigs := v.([]interface{})
 
 		for _, config := range httpLogsConfigs {
-			httpLogsConfig := config.(map[string]interface{})
-
 			logs.HTTPLogs = &web.HTTPLogsConfig{}
 
+			if config == nil {
```

Comment:
Since the `http_logs` is a list with maximum length set to `1`, the for loop here is a bit of weird. However, given this is the existing code, let's keep it as it was for now. 

Can we add the same logic for `application_logs`?

Reply:
It's less likely to occur there since you can turn those off via the log levels, but it wouldn't hurt. I'll update in the morning